app/views/find/v2/results/filters/_all.html.erb

```diff
@@ -0,0 +1,78 @@
+<div class="app-filter__content">
```

Comment:
I decided to create this partial because it was too big too follow on index. Let me know your opinions.